package.json

```diff
@@ -20,5 +25,5 @@
   "bugs": {
     "url": "https://github.com/auto-wcag/auto-wcag/issues"
   },
-  "homepage": "https://github.com/auto-wcag/auto-wcag#readme"
-}
+	"homepage": "https://github.com/auto-wcag/auto-wcag#readme"
+}
```

Comment:
EOF newline removed; I'd suggest we add an `.editorconfig` to prevent these inconsistencies from sneaking in.